docs/src/content/docs/components/code.mdx

```diff
@@ -85,14 +85,14 @@ You can then pass this imported string to the `<Code>` component to include it o
 # src/content/docs/example.mdx
 
 import { Code } from '@astrojs/starlight/components';
-import importedCode from '/src/env.d.ts?raw';
+import importedCode from '/tsconfig.json?raw';
```

Comment:
With the [changes to the TypeScript configuration](https://5-0-0-beta.docs.astro.build/en/guides/upgrade-to/v5/#changed-typescript-configuration), I removed all `env.d.ts` files so this example is no longer working (and relevant?). I switched to showing instead the default (and new) `tsconfig.json` file for Astro projects (we have a verbatim copy in the basics example).